eras/conway/test-suite/cardano-ledger-conway-test.cabal

```
@@ -75,6 +76,7 @@ test-suite cardano-ledger-conway-test
         cardano-ledger-allegra,
         cardano-ledger-alonzo,
         cardano-ledger-babbage,
+        cardano-ledger-babbage-test,
```

This package now requires functionality in `cardano-ledger-babbage`, so bounds need to be reflected accordingly
```suggestion
        cardano-ledger-babbage-test >=1.1.1 && <1.2,
```

Note, that if this dependency was in the library section too, then it would be sufficient to update in the library section only, but since it is not present there bounds need to be set here.